net.gpro.gproapp.yml

```diff
@@ -0,0 +1,94 @@
+app-id: net.gpro.gproapp
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
```

Comment:
why do you need this since you are not building from source?

(or better why aren't you building from source?)